webpack/InsightsCloudSync/Components/RemediationModal/RemediateButton.js

```diff
@@ -1,59 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { isEmpty } from 'lodash';
-import { Button, Popover } from '@patternfly/react-core';
+import { Button } from '@patternfly/react-core';
 import { translate as __ } from 'foremanReact/common/I18n';
-import { foremanUrl } from '../../../ForemanRhCloudHelpers';
 
-const RemediateButton = ({ isExperimentalMode, selectedIds, toggleModal }) => {
-  const [isVisible, setVisible] = React.useState(true);
-
-  const popoverContent = __(
-    `To use this feature, please enable <a href=${foremanUrl(
-      '/settings?search=name+%3D+lab_features'
-    )}>Show Experimental Labs</a> in settings.`
-  );
-
-  let button = (
-    <Button
-      variant="primary"
-      isDisabled={isEmpty(selectedIds)}
-      onClick={() => {
-        if (!isExperimentalMode) {
-          setVisible(value => !value);
-        } else {
-          toggleModal();
-        }
-      }}
-    >
-      {__('Remediate')}
-    </Button>
-  );
-
-  if (!isExperimentalMode) {
-    button = (
-      <Popover
-        isVisible={isVisible}
-        bodyContent={
-          <div dangerouslySetInnerHTML={{ __html: popoverContent }} />
-        }
-      >
-        {button}
-      </Popover>
-    );
-  }
-  return button;
-};
+const RemediateButton = ({ selectedIds, toggleModal }) => (
+  <Button
```

Comment:
Yay! this is much simpler now,
I would even suggest moving it to `RemediationModal.js` for simplicity and to avoid the extra file for a simple button